src/components/pages/LockScreen.tsx

```diff
@@ -32,6 +32,8 @@ const handleChange = async (
     : logger.error(
         `Error while calling toggleSetting('${setting}'), returned value is null or undefined."`
       )
+
+  return res
```

Comment:
Shouldn't we rely on the `dispatch` here?

Reply:
Initiallly yes, but for biometry I need the actual return value